src/cli/commands/init.js

```diff
@@ -136,6 +137,9 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
         }
       } else {
         answer = (await reporter.question(question)) || def;
+        if (entry.isShorthand && entry.isShorthand.exec(answer)) {
```

Comment:
It was to discern the repository case similar to the `If (entry.validation)` a couple lines up. I'll make the changes you requested layer today.